### DISCUSSION TITLE
XBox Controller Button bindings don't work unless OPENTRACK windows is active

### DISCUSSION THREAD

**Discussion by Airbumps** [opentrack/opentrack#1445] [Q&A]
Hi All,

Odd problem, been using open track for some time (2022.1.1) in MSFS and P3D without issue using regular joysticks and having various joystick buttons selected to STOP and START tracking. Its been working irrespective of the OpenTrack program being in the background and the Flightsim being in the forground.

I purchased an XBox controller for PC and assigned various buttons to STOP and START tracking. Whilst these buttons will be effective if the Opentrack program is selected/active/in the foreground, if its not the buttons are not recognised. So if for example I click on the open track window and press Y to START TRACKING it will indeed start tracking. If however I then click on the Windows 10 desktop and press X to STOP TRACKING, the button press is not recognised by OpenTrack. 

I've tried different USB ports and even a complete reinstall of Windows 10 - no chance.

Any ideas?
Cheers,

**Comment by sthalik** [comment]
This is normal, because XInput devices aren't yet supported.

**Reply by Airbumps** [reply]
Ok Thanks - That at least lets me know its not something wrong with my system.

Is there any expected implementation date for support of XInput devices? 

Cheers

**Reply by sthalik** [reply]
No ETAs. For now, you can use a program to convert button presses into keystrokes.
